powerhome/playbook
# Add input_options prop to Typeahead + Text Input [NUXE-644]

From the Playbook Kit example useage:

```erb
<%= pb_rails("typeahead", props: {
  label: "user",
  name: :foo,
  data: { typeahead_example: true },
  input_options: {
    classname: "my-typeahead-class",
    data: {
      typeahead_testing: "data field test"
    },
    id: "typeahead-input-id-test",
  },
})%>
```

#### Screens

<img width="1080" alt="Screen Shot 2021-08-05 at 12 23 37 PM" src="https://user-images.githubusercontent.com/2293844/128395060-95c2d5cc-dfd8-44ef-ac96-99f9ebc2b4df.png">

#### Breaking Changes

No

#### Runway Ticket URL

https://nitro.powerhrg.com/runway/backlog_items/NUXE-644
https://github.com/powerhome/playbook/issues/1360

#### How to test this

View the Rails example, verify that: 

- id
- class
- data

Attributes are rendered per the example src

#### Checklist:


*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*
